lib/Tmdb/HttpClient/Adapter/GuzzleAdapter.php

```diff
@@ -104,6 +104,7 @@ public function getConfiguration(Request $request)
         $this->request = $request;
 
         return [
+            'base_uri' => $request->getOptions()->get('base_url'),
```

Comment:
I guess `base_uri` here is a typo?

Reply:
It's actually the [correct name](http://docs.guzzlephp.org/en/stable/request-options.html?highlight=base_uri) (I remember re-reading the docs because it looked unusual to me too ;) )

Reply:
I see, just ran into the same confusion ;) Thanks for the follow-up. I'd like to check if this could be covered with test prior merging.